subworkflows/local/bam_merge_index_samtools/main.nf

```diff
@@ -9,30 +9,31 @@ include { SAMTOOLS_MERGE as MERGE_BAM       } from '../../../modules/nf-core/sam
 
 workflow BAM_MERGE_INDEX_SAMTOOLS {
     take:
-        bam // channel: [mandatory] meta, bam
```

Comment:
slightly opposed removing all the indents here :D , I like them for readability

Reply:
Like them too, but we need to be consistent